packages/story-editor/src/app/story/utils/getStoryPropsToSave.js

```diff
@@ -35,6 +35,9 @@ function getStoryPropsToSave({ story, pages, metadata }) {
     'autoAdvance',
     'defaultPageDuration',
     'backgroundAudio',
+    // TODO woof. need to make this dynamic based on
```

Comment:
```suggestion
    // TODO #8849. Need to make this dynamic based on
```

Reply:
ah yea. good call. the ticket id is a much better identifier than `woof`

Reply:
I liked the `woof`